lib/cosmos/tools/tlm_viewer/screen.rb

```diff
@@ -340,13 +340,15 @@ def process(filename_or_screen_def)
               # to the stack and that class doesn't have a complete method.
               current_widget.complete() if current_widget.respond_to? :complete
             when 'SETTING'
+              next unless current_widget
```

Comment:
current_widget is nil if you have a problem constructing it and then calling set_setting crashes